CachedImage.js

```diff
@@ -166,7 +166,7 @@ class CachedImage extends React.Component {
         const source = (this.state.isCacheable && this.state.cachedImagePath) ? {
             uri: 'file://' + this.state.cachedImagePath
         } : this.props.source;
-        if (this.props.fallbackSource && !this.state.cachedImagePath) {
+        if (this.props.fallbackSource && !this.state.isCacheable) {
```

Comment:
No, previous logic was correct. if `!this.state.cachedImagePath`, `source` would not be valid for being used as source below.